multiraft/multiraft.go

```diff
@@ -86,6 +86,17 @@ func (c *Config) validate() error {
 	return nil
 }
 
+// A GroupDeletedError is returned to a client for commands which are
+// pending while a group is deleted.
+type GroupDeletedError string
```

Comment:
I think it's more idiomatic go to use a value instead of a type for errors that don't need per-instance information: `var ErrGroupDeleted = errors.New("group deleted")`